### PR TITLE
slight optimisations

### PR DESCRIPTION
**Pull request by grosged**

### CODE REVIEW

**Comment by adriweb** [comment]
I'll merge that into a local branch so that travis can run the tests :)

**Comment by adriweb** [comment]
See here https://github.com/CE-Programming/toolchain/pull/116

**Comment by grosged** [comment]
ok thx, adriweb ;) (I can't understand this "Travis" at all, haha!..'m trying to ;) )

File: src/graphx/graphx.asm [hunk 1]
```diff
@@ -523,13 +525,15 @@ gfx_GetPixel:
 	inc	hl
 	inc	hl
 	inc	hl			; move to next argument
-	ld	de,0
 	ld	e,(hl)			; e = y coordinate
-	call	_PixelPtr
```
Comment (adriweb) [inline src/graphx/graphx.asm]: I believe this routine does some checks first, here is it safe to get rid of that?

**Comment by grosged** [comment]
oh, it's clearer  now, thx ;) I may guess why it would fail : I suggested not to check x,y coordinates when reading color of one pixel on screen)

**Comment by adriweb** [comment]
The graphx-related tests all ran fine, I'll let other people review :P

**Pull request merged**